res/values/strings.xml

```diff
@@ -1499,6 +1499,7 @@
     <string name="BackupDialog_disable_and_delete_all_local_backups">Disable and delete all local backups?</string>
     <string name="BackupDialog_delete_backups_statement">Delete backups</string>
     <string name="BackupDialog_copied_to_clipboard">Copied to clipboard</string>
+    <string name="BackupDialog_activation_error">Failed to activate backups. Please try again or contact support.</string>
```

Comment:
Let's keep our copy separate from Signal's copy and also keep the copy that still needs translation separate (i.e. put it under the "Next round of translation" mark)